packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -264,14 +234,16 @@ export const replaceElectorate = async (permittedPowers, config) => {
     },
   );
 
+  const governedContractKitsMap =
+    await permittedPowers.consume.governedContractKits;
   const psmKitMap = await permittedPowers.consume.psmKit;
 
+  console.log('RABI', [...governedContractKitsMap.values()]);
```

Comment:
meant to be dropped?

(I use 'CTH' (my initials) for the same purpose, and have a pre-commit script I run routinely to find phrases which should seldom be submitted.  YMMV.)